src/scripts/quests/questTypes/UsePokeballQuest.ts

```diff
@@ -11,7 +11,17 @@ class UsePokeballQuest extends Quest implements QuestInterface {
     }
 
     public static generateData(): any[] {
-        const possiblePokeballs = [GameConstants.Pokeball.Pokeball, GameConstants.Pokeball.Greatball, GameConstants.Pokeball.Ultraball];
+        const possiblePokeballs = [GameConstants.Pokeball.Pokeball];
+        if (player.highestRegion() > GameConstants.Region.kanto) {
+            possiblePokeballs.push(GameConstants.Pokeball.Greatball, GameConstants.Pokeball.Ultraball);
```

Comment:
Lavender town and Fuchsia city will always be unlocked when highestRegion is above kanto, right? We should just be able to remove this, and only use the code below?

Reply:
Yes, that would be true and we could do that, I included it as a small optimization.

Reply:
It's a very small optimization, compared to all the other code running all the time. It's not worth making the code more complex for

Reply:
Scrapped it 👍